src/lib/streamSpecToRegistry.ts

```diff
@@ -24,7 +24,14 @@ export default async function streamSpecToRegistry(spec: string) {
 
   debug('file and stream created, streaming into form data payload');
   const formData = new FormData();
-  formData.append('spec', stream);
+  formData.append('spec', {
```

Comment:
This work seems fine but `formdata-node` is still pretty shitty. Recommend moving off it and `node-fetch` as soon as we can. Left a diff in https://github.com/readmeio/rdme/issues/801#issuecomment-1720097752 with the work you can do to this file to move it over to all native APIs.